sos/plugins/memory.py

```diff
@@ -37,7 +37,7 @@ def setup(self):
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
-            "swapon --bytes --show"
+            "swapon --summary --verbose"
```

Comment:
What is 14.04 shipping that is so old it doesn't have this? Or is it using a non-util-linux implementation of swapon?